src/components/Receive.jsx

```diff
@@ -109,6 +110,16 @@ export default function Receive({ currentWallet }) {
                     'Copy'
                   )}
                 </rb.Button>
+                <input
+                  aria-hidden
+                  ref={inputRef}
+                  defaultValue={address}
+                  style={{
+                    position: 'absolute',
+                    left: '-9999px',
+                    top: '-9999px',
+                  }}
+                />
```

Comment:
```suggestion
                <input
                  readOnly
                  aria-hidden
                  ref={inputRef}
                  defaultValue={address}
                  style={{
                    position: 'absolute',
                    left: '-9999px',
                    top: '-9999px',
                  }}
                />
```

Stops the keyboard from popping up when selecting the text on mobile.